From eacd9880542485aeb47f9dd4cb0d8fdd0d1a5619 Mon Sep 17 00:00:00 2001
From: Haixia Shi <hshi@chromium.org>
Date: Wed, 22 Apr 2015 11:09:01 -0700
Subject: [PATCH 24/24] CHROMIUM: i915g: Always use X tiling for textures

This seems to achieve best performance after moving pineview-freon
to 3.8 kernel.

BUG=none
TEST=compile

Signed-off-by: Prince Agyeman <prince.agyeman@intel.com>
Signed-off-by: Dhinakaran Pandiyan <dhinakaran.pandiyan@intel.com>
Signed-off-by: James Ausmus <james.ausmus@intel.com>
Signed-off-by: Tomasz Figa <tfiga@chromium.org>
---
 src/gallium/drivers/i915/i915_resource_texture.c | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/src/gallium/drivers/i915/i915_resource_texture.c b/src/gallium/drivers/i915/i915_resource_texture.c
index 9a3279ccb755..bd0c67b37ffa 100644
--- a/src/gallium/drivers/i915/i915_resource_texture.c
+++ b/src/gallium/drivers/i915/i915_resource_texture.c
@@ -181,13 +181,7 @@ i915_texture_tiling(struct i915_screen *is, struct i915_texture *tex)
    if (tex->b.b.target == PIPE_TEXTURE_1D)
       return I915_TILE_NONE;
 
-   if (util_format_is_s3tc(tex->b.b.format))
-      return I915_TILE_X;
-
-   if (is->debug.use_blitter)
-      return I915_TILE_X;
-   else
-      return I915_TILE_Y;
+   return I915_TILE_X;
 }
 
 
-- 
2.5.1

